README.md

```diff
@@ -102,6 +103,19 @@ Currently everything should be compatible with **PHP 5.3-8.0**. _5.2 support is
 They may also contain some fixes, either for long-standing bugs, which haven't made their way into zf1 official repo before EOL, or newly found ones
 and (backwards compatible) adjustments (optimisations for composer autoloader mostly). Maybe even one or two new features.
 
+Still, the main purpose is to allow working on legacy projects on more modern systems, while opening the possibility to **migrate away from zf1 gradually, one component at a time**.
```

Comment:
@falkenhawk what do you think about adding `suggest` section including all `zf1s/*` packages?

Reply:
that might be an overkill 😅

Reply:
you're right, let's ignore it